diff --git a/node_modules/react-native-gesture-handler/lib/typescript/components/touchables/GenericTouchable.d.ts b/node_modules/react-native-gesture-handler/lib/typescript/components/touchables/GenericTouchable.d.ts
index 343288f..6c843e7 100644
--- a/node_modules/react-native-gesture-handler/lib/typescript/components/touchables/GenericTouchable.d.ts
+++ b/node_modules/react-native-gesture-handler/lib/typescript/components/touchables/GenericTouchable.d.ts
@@ -1,4 +1,3 @@
-/// <reference types="node" />
 import { Component } from 'react';
 import { StyleProp, ViewStyle, TouchableWithoutFeedbackProps } from 'react-native';
 import { GestureEvent, HandlerStateChangeEvent } from '../../handlers/gestureHandlers';
